src/framework-support/frameworks/corespring-legacy.js

```diff
@@ -6,6 +6,10 @@ export function support(dependencies) {
 
   if (isLegacy) {
     return {
+      externals: {
```

Comment:
can you add a comment that this is a temporary option only present to handle the `MathJax` dependency and that we plan to remove it.